examples/bench.rs

```diff
@@ -83,11 +83,9 @@ struct SpriteBundleDef {
 
 #[typetag::serde]
 impl ProtoComponent for SpriteBundleDef {
-    fn insert_self(&self, commands: &mut ProtoCommands, _asset_server: &Res<AssetServer>) {
+    fn insert_self(&self, commands: &mut ProtoCommands, asset_server: &Res<AssetServer>) {
         // === Get Prepared Assets === //
-        let texture: Handle<Image> = commands
-            .get_handle(self, &self.texture_path)
-            .expect("Expected Image handle to have been created");
+        let texture: Handle<Image> = asset_server.get_handle(&self.texture_path);
```

Comment:
By changing this line to get the handle from the `AssetServer` instead of the `ProtoCommands`, the bench time on my machine decreased to avg _20ms_, which is actually 5ms _faster_ than the average programmatic batch!

It might be worth investigating why getting the handle from `ProtoData` is slower.

Reply:
I imagine it's because getting the handle in `ProtoData` involves 3 lookups, whereas I believe `AssetServer` only needs one.